keyboards/tkc/portico68v2/portico68v2.c

```diff
@@ -0,0 +1,148 @@
+    {0,CS36_SW9,CS35_SW9,CS34_SW9}, /* RGB76 */
+    {0,CS39_SW1,CS38_SW1,CS37_SW1}, /* RGB77 */
+    {0,CS39_SW2,CS38_SW2,CS37_SW2}, /* RGB78 */
+    {0,CS39_SW3,CS38_SW3,CS37_SW3}, /* RGB79 */
+    {0,CS39_SW4,CS38_SW4,CS37_SW4}, /* RGB80 */
+    {0,CS39_SW5,CS38_SW5,CS37_SW5}, /* RGB81 */
+    {0,CS39_SW6,CS38_SW6,CS37_SW6}, /* RGB82 */
+};
+
+led_config_t g_led_config = {
+    {
+	    {  0,      1,  2,      3,      4,      5,  6,      7,      8,      9, 10, 11, 12,     13, 14 },
+        { 15,     16, 17,     18,     19,     20, 21,     22,     23,     24, 25, 26, 27,     28, 29 },
+        { 30,     31, 32,     33,     34,     35, 36,     37,     38,     39, 40, 41, 42, NO_LED, 43 },
+        { 44, NO_LED, 45,     46,     47,     48, 49,     50,     51,     52, 53, 54, 55,     56, 57 },
+        { 58,     59, 60, NO_LED, NO_LED, NO_LED, 61, NO_LED, NO_LED, NO_LED, 62, 63, 64,     65, 66 }
+    }, {
+        {   0,   0 }, {  15,   0 }, {  30,   0 }, {  45,   0 }, {  60,   0 }, {  75,   0 }, {  90,   0 }, { 105,   0 }, { 120,   0 }, { 135,   0 }, { 150,   0 }, { 165,   0 }, { 180,   0 }, { 203,   0 }, { 224,   0 },
+        {   4,  16 }, {  23,  16 }, {  38,  16 }, {  53,  16 }, {  68,  16 }, {  83,  16 }, {  98,  16 }, { 113,  16 }, { 128,  16 }, { 143,  16 }, { 158,  16 }, { 173,  16 }, { 188,  16 }, { 206,  16 }, { 224,  16 },
+        {   6,  32 }, {  26,  32 }, {  41,  32 }, {  56,  32 }, {  71,  32 }, {  86,  32 }, { 101,  32 }, { 116,  32 }, { 131,  32 }, { 146,  32 }, { 161,  32 }, { 176,  32 },               { 201,  32 }, { 224,  32 },
+        {   9,  48 },               {  34,  48 }, {  49,  48 }, {  64,  48 }, {  79,  48 }, {  94,  48 }, { 109,  48 }, { 124,  48 }, { 139,  48 }, { 154,  48 }, { 169,  48 }, { 189,  48 }, { 210,  48 }, { 224,  48 },
+        {   2,  64 }, {  21,  64 }, {  39,  64 },                                           {  96,  64 },                                           { 152,  64 }, { 171,  64 }, { 195,  64 }, { 210,  64 }, { 224,  64 },
+        {0,0},{180,0},{224,0},{200,64},{224,64},{224,32},{44,0},{88,0},{136,0},{0,32},{0,64},{24,64},{78,64},{112,64},{146,64}
+}, {
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,    4, 4,
+        4,    4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4,          4,          4, 4, 4, 4, 4,
+        2,2,2,2,2,2,2,2,2,2,2,2,2,2,2
+    }
+};
+
+void rgb_matrix_indicators_kb(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        for (uint8_t i = 0; i < DRIVER_1_LED_TOTAL; i++) {
+            rgb_matrix_set_color(i, 0xFF, 0xFF, 0xFF);
+        }
+    }
+    else {
+         if (rgb_matrix_get_flags() == LED_FLAG_NONE) {
+            for (uint8_t i = 0; i < DRIVER_1_LED_TOTAL; i++) {
+                rgb_matrix_set_color(i, 0, 0, 0);
+            }
+        }
+    }
+	if (!rgb_matrix_is_enabled()) {
+		rgb_matrix_driver.flush();
+    }
```

Comment:
This may not be needed.

Reply:
When I originally wrote the code for the RGB Matrix version of the original Portico PCB, it was needed if the RGB was disabled via keycode.

Reply:
well, that's what I mean. using the flags and such, you shouldn't need the flush, as that is still done by the core code.

Reply:
OK, updated.